pkg/storage/gcs/store.go

```diff
@@ -226,9 +236,17 @@ func (g *gcs) PutCRC(ctx context.Context, objectName string, reader io.Reader, d
 	return toSentinelErrors(err)
 }
 
-func (g *gcs) Delete(ctx context.Context, objectName string) (err error) {
+func (g *gcs) Delete(ctx context.Context, objectName string, opts ...storage.Option) (err error) {
```

Comment:
Similarly, we discussed the point about adding options.

i) this is not needed to address the requirement
ii) we don't want to generalize support for versioning, actually quite the opposite (labels stand out as an exception)
iii) adding options to a Store follows a different pattern, injecting options with `New()` rather than with every method

Reply:
yes, all `storage/gcs` changes are discussed and resolved in #419